charts/nginx-ingress/values.yaml

```diff
@@ -380,7 +380,7 @@ controller:
 
     ## The name of the service account of the Ingress Controller pods. Used for RBAC.
     ## Autogenerated if not set or set to "".
-    # name: nginx-ingress
+    name: nginx-ingress
```

Comment:
By setting this label we are forcing the name of the Service Account to be `nginx-ingress` right? This would mean that Service Accounts that were created using the autogenerated name would be deleted and a new one made with the name `nginx-ingress` right?

Reply:
the autogenerated name is `nginx-ingress`

Reply:
If I do a Helm deploy today the auto generated name is <release-name>-nginx-ingress

So for example if I do `helm install test-release ...` I get `test-release-nginx-ingress` as the name of the ServiceAccount.